resources/views/dashboard/admin/realops/index.blade.php

```diff
@@ -30,7 +30,7 @@
     <thead>
         <tr>
             <th scope="col">Date</th>
-            <th scope="col">Flight Number</th>
+            <th scope="col">Flight Number<br>Callsign</th>
```

Comment:
Do we wanna reverse this since it's from the pilot's perspective? Have the callsign on top and flight number underneath? Or maybe even just remove the flight number and just include what the pilot will file, and then leave the logo on the side the same to show the codeshare?

Reply:
Done.